runtime/server/generate_metrics_view.go

```diff
@@ -403,55 +395,32 @@ type metricsViewYAML struct {
 }
 
 type metricsViewDimensionYAML struct {
-	Label       string
-	Column      string
-	Description string
+	Label  string
+	Column string
 }
 
 type metricsViewMeasureYAML struct {
-	Name                string
-	Label               string
-	Expression          string
-	Description         string
-	FormatPreset        string `yaml:"format_preset"`
-	ValidPercentOfTotal any    `yaml:"valid_percent_of_total"`
+	Label       string
+	Expression  string
+	Name        string
+	Description string
 }
 
-func marshalMetricsViewYAML(doc *metricsViewYAML, aiPowered bool) (string, error) {
-	doc.AvailableTimeZones = []string{
-		"America/Los_Angeles",
-		"America/Chicago",
-		"America/New_York",
-		"Europe/London",
-		"Europe/Paris",
-		"Asia/Jerusalem",
-		"Europe/Moscow",
-		"Asia/Kolkata",
-		"Asia/Shanghai",
-		"Asia/Tokyo",
-		"Australia/Sydney",
-	}
-
-	doc.AvailableTimeRanges = []string{
-		"PT6H",
-		"PT24H",
-		"P7D",
-		"P14D",
-		"P4W",
-		"P3M",
-		"P12M",
-		"rill-TD",
-		"rill-WTD",
-		"rill-MTD",
-		"rill-QTD",
-		"rill-YTD",
-		"rill-PDC",
-		"rill-PWC",
-		"rill-PMC",
-		"rill-PQC",
-		"rill-PYC",
```

Comment:
Are these already the defaults in the UI? Or should we update the UI to use these as defaults if they are unset in the YAML?

Reply:
![image](https://github.com/rilldata/rill/assets/1181922/8a9d6267-2b83-47d3-b2ef-c9d04cf9b3f2)
Default ranges